lib/utils/isStandardSyntaxUrl.js

```diff
@@ -40,7 +40,7 @@ module.exports = function (url) {
 	// In url without quotes scss variable can be everywhere
 	// But in this case it is allowed to use only specific characters
 	// Also forbidden "/" at the end of url
-	if (url.includes('$') && /^[$\s\w+-/*'"/]+$/.test(url) && !url.endsWith('/')) {
+	if (url.includes('$') && /^[$\s\w+\-,./*'"]+$/.test(url) && !url.endsWith('/')) {
```

Comment:
See [regexp/no-obscure-range](https://ota-meshi.github.io/eslint-plugin-regexp/rules/no-obscure-range.html) and [regexp/no-dupe-characters-character-class](https://ota-meshi.github.io/eslint-plugin-regexp/rules/no-dupe-characters-character-class.html) - `[+-/]` means a range between between `+` (index 43) and `/` (index 47). It is the same as `[+,-./]`.